app/src/shared/components/Notifications/BasicNotification/basic.pcss

```diff
@@ -1,5 +1,7 @@
 .container {
   line-height: 28px;
+  display: flex;
+  align-items: center;
```

Comment:
This change to `display: flex` makes icon+text align naturally centered without manual adjustment of individual icon alignment.